api/service_handlers.go

```diff
@@ -175,7 +175,15 @@ func serviceUnbindApp(c echo.Context) error {
 		return err
 	}
 
-	if err = manager.UnbindApp(c.Request().Context(), c.Param("instance")); err != nil {
+	appName, err := formValue(c.Request(), "app-name")
+	if err != nil {
+		if appName == "" {
+			return c.NoContent(http.StatusBadRequest)
+		}
+		return err
+	}
```

Comment:
I'd drop this validation... It is already performed on RPaaS manager implementation.